chronograf.go

```diff
@@ -125,14 +125,15 @@ type Range struct {
 
 // TemplateValue is a value use to replace a template in an InfluxQL query
 type TemplateValue struct {
-	Value string `json:"value"`
-	Type  string `json:"type"`
+	Value    string `json:"value"`    // Value is the specific value used to replace a template in an InfluxQL query
+	Type     string `json:"type"`     // Type can be tagKey, tagValue, fieldKey, csv, measurement, database, constant
```

Comment:
In chronograf.go I wanted to just state the types but not too much (or any) logic.  However, at the ingestion point (in the API) it is its responsibility to verify the data is good.